### DISCUSSION TITLE
Snippets save location (Mac OS X)

### DISCUSSION THREAD

**Discussion by RomainLaf** [tahabasri/snippets#13] [Q&A] [answered]
Hello, 

I'd like to extract my snippets, saved on a Time Machine Backup (OSX) and I'd like to restore my snippets. 
Where can I find thoses snippets ? Where are they stored ? (Mac Os X) 

Thanks !

**Comment by tahabasri** [comment, marked as answer]
Snippets are located in `/Users/<username>/Library/Application Support/Code/User/globalStorage/tahabasri.snippets/`, next release should include an option to change that location to a custom one.
